src/helpers/configuration-validators.ts

```diff
@@ -1,7 +1,8 @@
 import { ErrorCode, PurchasesError } from "../entities/errors";
 
 export function validateApiKey(apiKey: string) {
-  if (!apiKey.startsWith("rcb_")) {
+  const api_key_regex = /^rcb_[a-zA-Z0-9_.-]+$/;
```

Comment:
Right now we don't allow `_`, `.` or `-`. But I thought to support it just in case... Happy to change that back though.

Reply:
We do support `_` for sandbox keys, which are `rcb_sb_...`